sample/src/main/java/co/lujun/sample/MainActivity.java

```diff
@@ -132,6 +139,19 @@ public void onTagCrossClick(int position) {
 
         // After you set your own attributes for TagView, then set tag(s) or add tag(s)
         mTagContainerLayout1.setTags(list1);
+        for (int i=0; i<list1.size(); i++) {
+            final int index = i;
+            Glide.with(mTagContainerLayout1.getContext())
+                    .asBitmap()
+                    .load("https://d1marr3m5x4iac.cloudfront.net/images/block/movies/17214/17214_aa.jpg")
```

Comment:
If the image is a big one, remember to ask Glide to resize it for you, so that this will get cached resized, which will improve performances

Reply:
Done!